wave_vision/tests/fast_tests.cpp

```diff
@@ -0,0 +1,115 @@
+// Libwave Headers
+#include "wave/wave_test.hpp"
+#include "wave/vision/fast_detector.hpp"
+
+namespace wave {
+
+#define TEST_CONFIG "tests/config/fast.yaml"
+#define TEST_IMAGE "tests/data/lenna.png"
+
+// Test fixture to load same image data
+class FASTTest : public testing::Test {
+ protected:
+    FASTTest() {}
+    virtual ~FASTTest() {
+        if (this->detector) {
+            delete this->detector;
+        }
+    }
+
+    void initDetector() {
+        this->detector = new FASTDetector(TEST_CONFIG);
+    }
+
+    virtual void SetUp() {
+        initDetector();
+        this->image = cv::imread(TEST_IMAGE, cv::IMREAD_COLOR);
+    }
+
+    cv::Mat image;
+    FASTDetector *detector;
+    std::vector<cv::KeyPoint> keypoints;
+};
+
+// Checks that correct configuration can be loaded
+TEST(FASTTests, GoodInitialization) {
+    FASTDetector detector(TEST_CONFIG);
+    SUCCEED();
```

Comment:
This was more to just confirm that the proper configuration can be loaded. Although...just realized this is redundant due to the "GoodManualConfiguration" test, which loads a proper configuration and then also checks it. Will remove this function.